<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calc</class>
 <widget class="QWidget" name="Calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_atan">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>atan</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_cos">
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="pushButton_sqrt">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButton_mod">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_7">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pushButton_minus">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_pow">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_acos">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton_x">
     <property name="minimumSize">
      <size>
       <width>85</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(76, 84, 108);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButton_dot">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButton_C">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(76, 84, 108);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_div">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButton_0">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="pushButton_ln">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" rowspan="9">
    <widget class="QCustomPlot" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_plus">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="yBeginLine">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Y: begin</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_9">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_sin">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="pushButton_log">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_1">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_6">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="pushButton_eq">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(76, 84, 108);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButton_5">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_square_close">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_8">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="pushButton_mult">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="inputLine">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>55</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_tan">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>tan</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButton_square_open">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLineEdit" name="yEndLine">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Y: end</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButton_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(63, 70, 89);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_asin">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 58, 74);
border-width: 1px;
border-radius: 7px;
color:rgb(174, 198, 255);
height: 25px;</string>
     </property>
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="xBeginLine">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhPreferNumbers</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>X: begin</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="xEndLine">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>X: end</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
